.travis.yml

```diff
@@ -77,9 +64,6 @@ stages:
   - name: test
     if: NOT tag =~ ^v
 
-  - name: test-flaky-db
-    - if: NOT tag =~ ^v
-      -
```

Comment:
That was certainly not right. Anyway, I want to remove that for now, we can add later.